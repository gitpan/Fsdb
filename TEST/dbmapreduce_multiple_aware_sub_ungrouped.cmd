prog='dbmapreduce'
args='-M -k experiment -f TEST/dbmapreduce_multiple_aware_sub.pl -C simple_reducer'
predecessor=dbmapreduce_multiple_aware_sub.cmd
cmp='diff -cb '
in=TEST/dbmapreduce_grouped_incorrectly.in
