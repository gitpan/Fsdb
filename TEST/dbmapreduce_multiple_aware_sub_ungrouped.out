#fsdb experiment n
ufs_mab_sys 2
ufs_rcp_real 2
#   | dbroweval  -b { my $count = 0; my $current_key = ""; @out_args = (-cols =>[qw(experiment n)]); }  {  if ($current_key ne _experiment) { if ($current_key ne "") { $ofref = [ $current_key, $count ] }; $count = 1; $current_key = _experiment; } else { $count++; $ofref = undef; };  } -e { $ofref = [ $current_key, $count ]; }
#   | dbmapreduce -M -k experiment -f TEST/dbmapreduce_multiple_aware_sub.pl -C simple_reducer
